modules/Hoot/tools/overpassQueryPanel.js

```diff
@@ -58,14 +58,16 @@ export default class OverpassQueryPanel {
             .text( 'Back' );
     }
 
-    overpassQueryPanel() {
+    async overpassQueryPanel() {
         // construct input section for user custom overpass queries
         this.overpassQueryContainer = this.form
             .select( '.wrapper div' )
             .insert( 'div', '.modal-footer' )
             .classed( 'button-wrap user-input', true );
 
-        let overpassQueryValue = '',
+        const defaultValue = await Hoot.api.getDefaultOverpassQuery();
```

Comment:
how come we need defaultValue and overpassQueryValue?

Reply:
Once you written something custom in the text area overpassQueryValue becomes that.  The placeholder will stay the default value.  I've found if you make an edit and can ctrl+z undo it, you can close the pull dialog and the default query will be restored as the editable value.